tools/cli/adminClusterCommands.go

```diff
@@ -110,7 +110,9 @@ func AdminRebalanceStart(c *cli.Context) {
 }
 
 func AdminRebalanceList(c *cli.Context) {
-
+	c.Set(FlagWorkflowID, failovermanager.RebalanceWorkflowID)
+	c.GlobalSet(FlagDomain, common.SystemLocalDomainName)
```

Comment:
nit: not sure if we want to increase the retention period of the system so that the history for managed failover or rebalancing workflow can be stored for a longer period of time.

Reply:
Good point. I will create a task to track this.

Reply:
it is already 30 days.

Reply:
Interesting. Maybe we manually updated it internally. The retention specified in the schema is 3 days I think. 
See: https://github.com/uber/cadence/blob/master/schema/cassandra/cadence/schema.cql#L447